content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -1,11 +1,9 @@
 # Persisting Experiments
 
-DVC runs experiments outside of the Git stage/commit cycle for quick iteration.
-When your experiments are good enough to save or share, you may want to store
-them persistently as Git commits in your repository.
-
-In this section, we describe how to bring them to the standard Git workflow with
-`dvc exp branch` and `dvc exp apply`.
+DVC Experiments run outside of the regular Git workflow for faster iteration and
```

Comment:
if we start with DVC experiments as first entry point to experiments people won't know about "Git" commit based experiments. Phrases like `run outside of the regular Git workflow` to some extent are fine for the former DVC users but will be confusing for new users.

Reply:
> if we start with DVC experiments as first entry point to experiments

[This document](https://dvc-org-exp-dvc-exps-pa-qhfdse.herokuapp.com/doc/user-guide/experiment-management/persisting-experiments) is not a first entry point to the topic of experiments though. That's why there's an [index page](https://dvc-org-exp-dvc-exps-pa-qhfdse.herokuapp.com/doc/user-guide/experiment-management) and an [overview page](https://dvc-org-exp-dvc-exps-pa-qhfdse.herokuapp.com/doc/user-guide/experiment-management/experiments-overview) (WIP).

Reply:
> Phrases like "run outside of the regular Git workflow" ... will be confusing for new users

This could be more emphasized in the overview page though, so it makes sense here (updated in 73175a99 for now).